jukebox_radio/streams/models/queue_interval_model.py

```diff
@@ -69,9 +77,13 @@ def highest_bound(self, queue_id):
 
     def conflicting_interval(self, *, queue_id, lower_bound_id, upper_bound_id):
         Marker = apps.get_model('streams', 'Marker')
+        Queue = apps.get_model('streams', 'Queue')
+
+        queue = Queue.objects.select_related("track").get(uuid=queue_id)
+        track = queue.track
 
-        lower_bound = Marker.objects.get(uuid=lower_bound_id, queue_id=queue_id)
```

Comment:
This was wrong before - there is no queue field on Marker. We need to get the track from queue and use it to query for the bound markers.